Makefile

```diff
@@ -39,7 +39,7 @@ style:
 # run tests for the repo
 test:
 	@echo "Running python tests";
-	pytest tests $(PYTEST_ARGS)
+	pytest -r tests $(PYTEST_ARGS) --ignore tests/lmeval
```

Comment:
Shouldn't there be one or more letters after the `-r` flag to signify? I’ve only ever used this to include more reasons in the summary, since the default is for failures, but in CI we often want to see reasons for skipped tests as well. We typically use `-ra` for CI purposes in this regard.

From `pytest --help`:

```
  -r chars              show extra test summary info as specified by chars:
                        (f)ailed, (E)rror, (s)kipped, (x)failed, (X)passed,
                        (p)assed, (P)assed with output, (a)ll except passed
                        (p/P), or (A)ll. (w)arnings are enabled by default
                        (see --disable-warnings), 'N' can be used to reset
                        the list. (default: 'fE').
```